### PR TITLE
PS-678 : Databox admin - integration - adding yaml and Json validator

### PR DESCRIPTION
**Pull request by aynsix**

### CODE REVIEW

File: databox/api/config/validator/validation.yaml [hunk 1]
```diff
@@ -175,15 +175,15 @@ App\Entity\Core\Tag:
...
-    - App\Validator\ValidIntegrationOptionsConstraint: ~
-  properties:
-    optionsJson:
-      - Json:
-          message: You've entered an invalid Json.
-    optionsYaml:
-      - App\Validator\YamlConstraint: ~
+# App\Entity\Integration\WorkspaceIntegration:
+#   constraints:
+#     - App\Validator\ValidIntegrationOptionsConstraint: ~
```
Comment (4rthem) [inline databox/api/config/validator/validation.yaml:180]: Why removing this one?

File: databox/api/config/validator/validation.yaml [hunk 2]
```diff
@@ -175,15 +175,15 @@ App\Entity\Core\Tag:
...
+#   properties:
+#     optionsJson:
+#       - Json:
+#           message: You've entered an invalid Json.
+#     optionsYaml:
+#       - App\Validator\YamlConstraint: ~
```
Comment (4rthem) [inline databox/api/config/validator/validation.yaml:186]: Don't comment, just remove/clean. Git history is here to help!

File: lib/php/admin-bundle/Form/JsonType.php [hunk 3]
```diff
@@ -24,14 +26,31 @@ public function transform(mixed $value)
 
     public function reverseTransform(mixed $value)
     {
+        if ($value !== null && json_validate($value) === false) {
+            return ['input-error' => 'Invalid JSON: ' . json_last_error_msg()];
```
Comment (4rthem) [inline lib/php/admin-bundle/Form/JsonType.php:30]: I believe `{"input-error": "Invalid JSON: ..."}` will appear in the field?
The user must see its first (errored) input to correct it.

**Review by 4rthem — CHANGES_REQUESTED** [review]

File: lib/php/admin-bundle/Form/JsonType.php (hunk 3 above)
Comment (aynsix) [reply lib/php/admin-bundle/Form/JsonType.php:30]: No, it's in the reverseTransform, data will be insert to DB, the user input not change

**Pull request merged**